prompting/miners/hf_miner.py

```diff
@@ -50,28 +49,42 @@ def __init__(self, config=None):
         super().__init__(config=config)
 
         model_kwargs = None
-        if self.config.neuron.load_quantized:
-            bt.logging.info("Loading quantized model...")
+        if self.config.neuron.load_in_8bit:
```

Comment:
can we not also push this common logic (shared with agents) into a shared place?